app/views/report/bulk.html.erb

```diff
@@ -138,23 +131,14 @@ ul li{
   <span class="bulk_button" id="confirm-refresh-metadata-button">Refresh metadata</span><br>
 </div>
 
-<div class="bulk_operation" id="tag">
-  <h1>Change tags</h1>
-  <p>This needs a list of pids and tags, with a tab between the pid and each tag. E.g.:<br>
-  druid:bc088fn5010&lt;tab&gt;Project : Something&lt;tab&gt;Process : Content Type : Book
-  </p>
-  <p>This formatting is easier to perform by using Excel or a text editor, and then copying the result to this text area.</p>
-  <textarea style="width:50%" id="tags" name="tags"></textarea><br>
-  <button class="bulk_button" id="set_tags" name="set_tags">Update tags</button>
-</div>
 <div class="bulk_operation" id="source_id">
   <h1>Change source id</h1>
   <p>This needs a list of pids and source ids, with a tab between the pid and the corresponding source id. E.g.:<br>
   druid:bc088fn5010  fuller:M1090_S15_B02_F12_0007
   </p>
   <p>This formatting is easier to perform by using Excel or a text editor, and then copying the result to this text area.</p>
   <textarea style="width:50%" id="source_ids" name="source_ids"></textarea><br>
-  <span class="bulk_button" id="set_source_id" name="set_source_id">Update source ids</span>
+  <button class="bulk_button" id="set_source_id" name="set_source_id">Update source ids</button>
```

Comment:
Does this affect the UI? Or is it for Capybara purposes?

Reply:
Yes, the test fails with `span` because of "click_button".

Reply:
OK, great!